### DISCUSSION TITLE
How to enable click on Macbook Silicon M2?

### DISCUSSION THREAD

**Discussion by solvorry** [Genymobile/feedback#81] [Genymotion Desktop] [answered]
When install fresh it works but when I restart machine and then run Genymotion then click on Genymotion devices is gets disabled and not able to click on Genymotion device.

Kindly help me someone who resolved the issue.

**Comment by jmaneyrol69** [comment, marked as answer]
Hi @solvorry,
A solution to this problem has been found. See https://support.genymotion.com/hc/en-us/articles/19740932095133 for detailed instructions.
